docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/snippets/basemanifest.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-microservice
+  labels:
+    app: my-microservice
+  annotations:
+    source: base-manifest-test
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: my-microservice
+  template:
+    metadata:
+      labels:
+        app: my-microservice
+    spec:
+      serviceAccountName: my-microservice
+      volumes:
+        - name: secret-vol
+          secret:
+            secretName: my-microservice-secret0
+      containers:
+        - name: my-microservice
+          ports:
+          - containerPort: 80
+            protocol: TCP
+          volumeMounts:
+          - name: secret-vol
+            readOnly: true
+            mountPath: /etc/secret-vol
+          env:
+          - name: TEST_SECRET_KEY
+            valueFrom:
+              secretKeyRef:
+                name: my-microservice-secret1
+                key: secret1
+          - name: TEST_CONFIGMAP_KEY
+            valueFrom:
+              configMapKeyRef:
+                name: my-microservice-config
+                key: TEST_CONFIGMAP
```

Comment:
These resources aren't defined in the file. I'd remove them if they're not needed